YouTrackAutomation/Functions/New-YTIssue.ps1

```diff
@@ -0,0 +1,60 @@
+
+function New-YTIssue
+{
+    <#
+    .SYNOPSIS
+    Creates a new issue in YouTrack.
+
+    .DESCRIPTION
+    The `New-YTIssue` function creates a new issue in YouTrack. Pass the short name of the project where the issue

```

Comment:
Update: "Pass the project's short name, ID, or a project object where the issue should get created to the `Project` parameter."